### ISSUE TITLE
Issue with the innodb_log_file_size parameter

### ISSUE THREAD

**Issue by deviantony** [puppetlabs/puppetlabs-mysql#392]
If you try to apply a new log file size to the innodb_log_file using for example :

``` puppet
class { '::mysql::server':
    override_options => {
        'innodb_log_file_size' => '128M',
        ...
    }
}
```

The puppet module will fail to start the mysql service :

```
Error: /Stage[main]/Mysql::Server::Service/Service[mysqld]: Failed to call refresh: Could not start Service[mysqld]: Execution of '/sbin/start mysql' returned 1: 
Error: /Stage[main]/Mysql::Server::Service/Service[mysqld]: Could not start Service[mysqld]: Execution of '/sbin/start mysql' returned 1: 
```

And in the mysql error log :

```
Completed initialization of buffer pool
InnoDB: Error: log file ./ib_logfile0 is of different size 0 5242880 bytes
InnoDB: than specified in the .cnf file 0 134217728 bytes!
```

Because this parameter needs a certain way to handle a new value, before you can apply to a new mysql server, you need to remove the $MYSQL_DATA_DIR/ib_logfile0 and $MYSQL_DATA_DIR/ib_logfile1 before starting to restart the MySQL service.

**Comment by Aaron-Rob** [comment]
Hello! We are doing some house keeping and noticed that this issue has been open for a long time. We're going to close it but please do raise another issue if the issue still persists. :smile:
